### PR TITLE
Add option to configure SortedImportsRule to sort @testable imports on the top or on the bottom

### PR DESCRIPTION
**Pull request by MaxHaertwig**
Hi :)

What do you think about this configuration option? I'm open to rename the options `top` and `bottom` to `first` and `last`.

### CODE REVIEW

File: Tests/SwiftLintFrameworkTests/SortedImportsRuleTests.swift [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import SwiftLintFramework
+import XCTest
+
+class SortedImportsRuleTests: XCTestCase {
+    func testWithDefaultConfiguration() {
+        verifyRule(FileLengthRule.description, commentDoesntViolate: false,
```
Comment (marcelofabri) [inline Tests/SwiftLintFrameworkTests/SortedImportsRuleTests.swift]: can you just use `verifyRule(SortedImportsRule.description)`?

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,7 +10,9 @@
 
 #### Enhancements
 
-* None.
+* Add option to configure SortedImportsRule to sort `@testable` imports on
```
Comment (marcelofabri) [inline CHANGELOG.md]: this is now in the wrong section

Comment (marcelofabri) [reply CHANGELOG.md]: this should be in the "enhancements" section 😬

**Comment by bsarrazin** [comment]
@marcelofabri @MaxHaertwig I'm very much looking forward to see this merged 😝
Any progress since Feb 8th?

**Comment by MaxHaertwig** [comment]
@marcelofabri @jpsim I rebased my code on top of master. Can we get this merged?

**Comment by bsarrazin** [comment]
@jpsim @marcelofabri friendly bump

**Comment by bsarrazin** [comment]
My apologies for being annoying. @jpsim @marcelofabri

**Comment by bsarrazin** [comment]
Is it just me or it seems to take forever to merge things that are approved?
All checks are green, the PR is approved... who's responsible/authorized to merge this?

**Comment by MaxHaertwig** [comment]
@bsarrazin I don't know what's blocking this right now. You could use your own fork of SwiftLint with this PR merged in the meantime.

**Comment by bsarrazin** [comment]
> You could use your own fork of SwiftLint with this PR merged in the meantime.
the value isn't worth the trouble imo

Will just have to wait until this is merged. Maybe we can blame 2020? 😝

**Comment by bsarrazin** [comment]
Stale bot closing the PR that was passing CI and was green all around just because no one ever looked at it?

**Pull request closed without merging**